backend/game/player/player.py

```diff
@@ -32,3 +38,21 @@ def update_energy_by(self, change_integer):
         self.energy += change_integer
         if self.energy < constants.DEFAULT_ENERGY_CUBE:
             self.energy = constants.DEFAULT_ENERGY_CUBE
+
+    def add_card(self, card):
+        self.card_on_hand[card.name] = card.effect
```

Comment:
Not sure what the purpose of storing the card by its name and string of the effect in a dictionary like this

Reply:
see comments within backend/game/player/player.py above...